### DISCUSSION TITLE
May I ask if nvm-window software can be used commercially? Mainly intended for use by company employees.

### DISCUSSION THREAD

**Discussion by yt46767** [coreybutler/nvm-windows#1003] [Support]
May I ask if nvm-window software can be used commercially? Mainly intended for use by company employees.
 请问下nvm-window软件可以商业使用？主要想用于公司员工使用。

**Comment by coreybutler** [comment]
NVM for Windows is free for everyone, so it can be used by employees/employers. In fact, our download logs suggest the entire Fortune 500 uses NVM for Windows.

**Reply by dagilleland** [reply]
Also, @coreybutler, don't forget to mention your work on **Runtime** (I am really excited to see someone taking on the wonderful idea you have there!).

So, @yt46767 - Check out the updated notice on this repo's `README.md` and sign up for the Runtime beta!

(btw Corey - I hope you don't mind me hyping up your new venture 😉 )

**Reply by coreybutler** [reply]
@dagilleland Mind? I'm thrilled for the hype! Thank you!

More information is available at https://github.com/coreybutler/nvm-windows/wiki/Runtime.

**Comment by yt46767** [comment]
来信收到了哦~~~~
